packages/siop-oid4vp/lib/op/OP.ts

```diff
@@ -154,19 +158,47 @@ export class OP {
     }
   }
 
+  public static async extractEncJwksFromClientMetadata(clientMetadata: JwksMetadataParams) {
+    // The client metadata will be parsed in the joseExtractJWKS function
+    const jwks = await extractJwksFromJwksMetadata(clientMetadata)
+    const encryptionJwk = jwks?.keys.find((key) => key.use === 'enc')
```

Comment:
Probably wise to add a TODO for potential error handling or a warning if multiple JWKs with 'enc' usage are found